.travis.yml

```diff
@@ -14,9 +14,20 @@ env:
 
 before_install:
   # Docker-compose installation
-  - curl -L https://github.com/docker/compose/releases/download/1.8.1/docker-compose-`uname -s`-`uname -m` > docker-compose
-  - chmod +x docker-compose
-  - sudo mv docker-compose /usr/local/bin
+  - sudo apt-get update
+  - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
+  - docker-compose --version
```

Comment:
ok @ruflin  as u suggested I reverted some old code which installed a specific version of docker-compose I just upgraded to a more recent version, and I find a way to update the docker engine to a fixed version *1.13.x.* 
This would allow us to use in future docker-compose file version **3.0** and **3.2**